docs/src/linux.md

```diff
@@ -26,6 +26,7 @@ Zed is open source, and [you can install from source](./development/linux.md).
 There are several third-party Zed packages for various Linux distributions and package managers, sometimes under `zed-editor`. You may be able to install Zed using these packages:
 
 * Homebrew: [`zed`](https://formulae.brew.sh/cask/zed), [`zed@preview`](https://formulae.brew.sh/cask/zed@preview)
+* ALT Linux (Sisyphus): [`zed`](https://packages.altlinux.org/en/sisyphus/srpms/zed/)
```

Comment:
Can we add this to the bottom of the list (after Manjaro)?

Reply:
I added the sorting according to alphabetical order, but I think the position can be changed at your discretion.